.github/workflows/docker-build-push.yml

```diff
@@ -35,7 +35,10 @@ jobs:
         id: meta
         uses: docker/metadata-action@v5
         with:
-          images: ${{ matrix.image }}         
+          images: ${{ matrix.image }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3

```

Comment:
Can you explain this?

Reply:
https://github.com/marketplace/actions/build-and-push-docker-images

It's to allow building against more platforms